File: public/phase-2-20160620022203/JAVA/happy-colleagues.java
// Name : Happy Colleagues
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// r :  the number of candies of red
		// g :  the number of candies of green
		// b :  the number of candies of blue
				Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int r = Integer.parseInt(elements[0]);
		int g = Integer.parseInt(elements[1]);
		int b = Integer.parseInt(elements[2]);


		// write your code here
		// store your results in `number_of_colleagues`

		// output
		// Dummy Data
		int number_of_colleagues=4;

		System.out.println(number_of_colleagues);

	}
}

File: public/phase-2-20160620022203/C_SHARP/happy-colleagues.cs
// Name : Happy Colleagues
using System;
namespace Application {
	class happy_colleagues {
		public static void Main(string[] args) {
		// input:
		// r :  the number of candies of red
		// g :  the number of candies of green
		// b :  the number of candies of blue
				string[] elements = (Console.ReadLine()).Split(' ');
		int r = Convert.ToInt32(elements[0]);
		int g = Convert.ToInt32(elements[1]);
		int b = Convert.ToInt32(elements[2]);


		// write your code here
		// store your results in `number_of_colleagues`

		// output
		// Dummy Data
		int number_of_colleagues=4;

		Console.WriteLine(number_of_colleagues);

		}	
	}
}